app/models/timeline.rb

```diff
@@ -60,7 +60,7 @@ def standardize_homepage
 
     manifest_json = JSON.parse(manifest)
     manifest_json["homepage"] ||= {}
```

Comment:
Does this initialization need to change as well?
`manifest_json["homepage"] ||= []`

Reply:
Not sure if it's strictly necessary, since it didn't seem to interfere with the tests but probably best to make the change. Will switch it over.